### PR TITLE
Fix some Britishisms in the JSON files.

### PR DESCRIPTION
**Pull request by FeepingCreature**
#### Summary
Bugfixes "Fix Britishisms in the JSON files."

#### Purpose of change

In penance for being a bit annoying on the IRC, somebody requested that I do a review for britishisms. 

#### Describe the solution

Instead of reviewing thousands of lines manually, I got AI Claude 3.5 Sonnet to [make me a tool](https://gist.github.com/FeepingCreature/76bac93a7bff70cf5e8963c1f9cf23c4) that basically just scans through all given files word-by-word and flags (and, if requested, substitutes) words that are in the British (`en_UK`) but not American (`en_US`)system dictionary. 

#### Testing
Afterwards I did a manual review using `git diff --word-diff=color`, and consulted Merriam-Webster where unsure. Unless I've missed something, all words replaced *at least* list as "less common" on https://www.merriam-webster.com/ .

#### Additional context

Should I do `src` too? It's a bit riskier cause of autoserialization.

Also, should "arse" be dropped from the list of generic insults?

edit: An `"id"` change crept past me. I've grepped the diff; no `"id"` changes aside from that one.

### CODE REVIEW

**Comment by Maleclypse** [comment]
This comes up on occasion, please only fix player visible British spellings. Anything with // or in other comment methods is to stay as is.

**Comment by FeepingCreature** [comment]
Aight, should be fixed.

File: data/json/npcs/exodii/exodii_merchant_talk.json [hunk 1]
```diff
@@ -623,7 +623,7 @@
     "type": "talk_topic",
     "dynamic_line": {
       "//~": "This one can't be saved too much brain rotted but i can salvage bits from them or repurpose them for walker, or just cut out rot and neural interface so you can bury them proper",
-      "str": "This 'unwon't be twitching', too much grey lost to dead.  But ye can salvage bits from 'em or born 'em for a walker, or just cut out the rot 'n grey link so ya can miss 'em proper."
+      "str": "This 'unwon't be twitching', too much gray lost to dead.  But ye can salvage bits from 'em or born 'em for a walker, or just cut out the rot 'n gray link so ya can miss 'em proper."
```
Comment (Maleclypse) [inline data/json/npcs/exodii/exodii_merchant_talk.json:626]:
```suggestion
      "str": "This 'unwon't be twitching', too much grey lost to dead.  But ye can salvage bits from 'em or born 'em for a walker, or just cut out the rot 'n grey link so ya can miss 'em proper."
```
This character is canonically British adjacent.

File: data/json/npcs/exodii/exodii_merchant_talk.json [hunk 2]
```diff
@@ -1265,7 +1265,7 @@
     "type": "talk_topic",
     "dynamic_line": {
       "//~": "Spread, yes, through the portals, but we don't know how.  After the world is down to dust, I'm long gone.  Once or twice we've send back a probe to the deadlands we left behind, but none have returned.  As far as I know, there's only one way it goes from here.",
-      "str": "Spread, aye, through the wibbles, but we ain't ken on how.  After the green an' brown be to grey and black, so t'speak, Rubik be long gone.  On a moment, we folk've sent back a lover's robe or so t'the deadlands what left, but narry have returned.  An I'm ken, there be only one course from here."
+      "str": "Spread, aye, through the wibbles, but we ain't ken on how.  After the green an' brown be to gray and black, so t'speak, Rubik be long gone.  On a moment, we folk've sent back a lover's robe or so t'the deadlands what left, but narry have returned.  An I'm ken, there be only one course from here."
```
Comment (Maleclypse) [inline data/json/npcs/exodii/exodii_merchant_talk.json:1268]:
```suggestion
      "str": "Spread, aye, through the wibbles, but we ain't ken on how.  After the green an' brown be to grey and black, so t'speak, Rubik be long gone.  On a moment, we folk've sent back a lover's robe or so t'the deadlands what left, but narry have returned.  An I'm ken, there be only one course from here."
```

File: data/json/npcs/exodii/exodii_merchant_talk.json (hunk 1 above)
Comment (Psithief) [reply data/json/npcs/exodii/exodii_merchant_talk.json]: An American transcribing what he says would still write `gray`.

Comment (Maleclypse) [reply data/json/npcs/exodii/exodii_merchant_talk.json]: It's good it's dialogue and not transcribed then.

Comment (FeepingCreature) [reply data/json/npcs/exodii/exodii_merchant_talk.json]: I mean, I can see the argument: technically it's "transcribed" by the user interface. I guess it depends on whether the text we read is what the NPC says ("grey") or what our character hears ("gray").

**Pull request merged**